10-angular/custom_directives/readme.md

```diff
@@ -419,10 +426,11 @@ Now I need to make that grumble available inside the partial's HTML. I do this b
 
 ```js
 (function(){
-  var directives = angular.module('grumbleDirectives',[]);
-  directives.directive('grumbleShow', function(){
+  angular
+  .module("grumbles")
+  .directive("myCustomDirective", function(){
```

Comment:
also, line 417 above, we need to make sure that the HTML we pass into the template `_grumble_show.html`
is wrapped inside one element, like a `div`. Otherwise, we get this error:
https://docs.angularjs.org/error/$compile/tplrt?p0=grumbleShow&p1=js%2Fgrumbles%2F_grumble_show.html